core/wallet/transaction.go

```diff
@@ -467,19 +447,224 @@ func UpdateStatus(d ds.Datastore, peerId string, txId int64, status string) erro
 	return nil
 }
 
-func GetTransactions(d ds.Datastore, peerId string) ([]*walletpb.Transaction, error) {
+func GetTransactions(d ds.Datastore, peerId string) ([]*walletpb.TransactionV1, error) {
+	txs := make(TxSlice, 0)
+	v0Txs, err := loadV0Txs(d, peerId)
```

Comment:
Do you have plan to migrate from v0 to v1, so no more loading v0 in the future?

Reply:
yep, will migrate v0/v1 to v2(maybe) after offline-2-offline transfer ticket is done.